test/machines/awsclient.go

```diff
@@ -27,35 +27,60 @@ func (client *awsClientWrapper) GetRunningInstances(machine *clusterv1alpha1.Mac
 }
 
 func (client *awsClientWrapper) GetPublicDNSName(machine *clusterv1alpha1.Machine) (string, error) {
-	runningInstances, err := machineutils.GetRunningInstances(machine, client.client)
+	instance, err := machineutils.GetInstance(machine, client.client)
 	if err != nil {
 		return "", err
 	}
 
-	if len(runningInstances) == 0 {
-		return "", fmt.Errorf("no running machine instance found")
-	}
-
-	if *runningInstances[0].PublicDnsName == "" {
+	if *instance.PublicDnsName == "" {
```

Comment:
Is there a use case where it make sense to get public DNS name of an instance that is not running?

Reply:
`GetInstance` also just returns a running instance. I'll rename the method to `GerRunningInstance`.